specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7837,179 +7837,7 @@
-            "Standard_DS12_v2",
-            "Standard_DS13_v2",
-            "Standard_DS14_v2",
-            "Standard_DS15_v2",
-            "Standard_DS13-4_v2",
-            "Standard_DS13-2_v2",
-            "Standard_DS14-8_v2",
-            "Standard_DS14-4_v2",
-            "Standard_E2_v3",
-            "Standard_E4_v3",
-            "Standard_E8_v3",
-            "Standard_E16_v3",
-            "Standard_E32_v3",
-            "Standard_E64_v3",
-            "Standard_E2s_v3",
-            "Standard_E4s_v3",
-            "Standard_E8s_v3",
-            "Standard_E16s_v3",
-            "Standard_E32s_v3",
-            "Standard_E64s_v3",
-            "Standard_E32-16_v3",
-            "Standard_E32-8s_v3",
-            "Standard_E64-32s_v3",
-            "Standard_E64-16s_v3",
-            "Standard_F1",
-            "Standard_F2",
-            "Standard_F4",
-            "Standard_F8",
-            "Standard_F16",
-            "Standard_F1s",
-            "Standard_F2s",
-            "Standard_F4s",
-            "Standard_F8s",
-            "Standard_F16s",
-            "Standard_F2s_v2",
-            "Standard_F4s_v2",
-            "Standard_F8s_v2",
-            "Standard_F16s_v2",
-            "Standard_F32s_v2",
-            "Standard_F64s_v2",
-            "Standard_F72s_v2",
-            "Standard_G1",
-            "Standard_G2",
-            "Standard_G3",
-            "Standard_G4",
-            "Standard_G5",
-            "Standard_GS1",
-            "Standard_GS2",
-            "Standard_GS3",
-            "Standard_GS4",
-            "Standard_GS5",
-            "Standard_GS4-8",
-            "Standard_GS4-4",
-            "Standard_GS5-16",
-            "Standard_GS5-8",
-            "Standard_H8",
-            "Standard_H16",
-            "Standard_H8m",
-            "Standard_H16m",
-            "Standard_H16r",
-            "Standard_H16mr",
-            "Standard_L4s",
-            "Standard_L8s",
-            "Standard_L16s",
-            "Standard_L32s",
-            "Standard_M64s",
-            "Standard_M64ms",
-            "Standard_M128s",
-            "Standard_M128ms",
-            "Standard_M64-32ms",
-            "Standard_M64-16ms",
-            "Standard_M128-64ms",
-            "Standard_M128-32ms",
-            "Standard_NC6",
-            "Standard_NC12",
-            "Standard_NC24",
-            "Standard_NC24r",
-            "Standard_NC6s_v2",
-            "Standard_NC12s_v2",
-            "Standard_NC24s_v2",
-            "Standard_NC24rs_v2",
-            "Standard_NC6s_v3",
-            "Standard_NC12s_v3",
-            "Standard_NC24s_v3",
-            "Standard_NC24rs_v3",
-            "Standard_ND6s",
-            "Standard_ND12s",
-            "Standard_ND24s",
-            "Standard_ND24rs",
-            "Standard_NV6",
-            "Standard_NV12",
-            "Standard_NV24"
-          ],
-          "x-ms-enum": {
-            "name": "VirtualMachineSizeTypes",
```

Comment:
Pls. add a comment that this list of size constants is not kept up to date and the user should query the api https://docs.microsoft.com/en-us/rest/api/compute/resourceskus/list to get a list of all latest sizes available for their subscription in a region and add a note that these string constants will be removed from subsequent swagger spec versions.

Reply:
Updated the hyperlinks and added a note abut deprecating the string constants.